source/features/blocks/components/BlockList.scss

```diff
@@ -0,0 +1,87 @@
+.blockListContainer {
+  width: 872px;
```

Comment:
Alphabetic ordering

Reply:
This is WIP pr. Need to update several components, will do this sorting once they are done.